bucket/xmlnotepad.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.8.0.52",
+    "description": "Provides a simple intuitive User Interface for browsing and editing XML documents",
+    "homepage": "https://microsoft.github.io/XmlNotepad/",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/XmlNotepad/releases/download/2.8.0.52/XmlNotepadPackage_2.8.0.52_AnyCPU.msixbundle#/dl.7z",
+    "hash": "164f881fd86f9c3a35e111deb407b098c556cdca0ecd98aa57c8ec7083a60303",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\XmlNotepadPackage_2.8.0.52_AnyCPU.msix\" \"$dir\" -ExtractDir 'Application'",
```

Comment:
This will not work on next update. Use something like `XmlNotepadPackage_*_AnyCPU.msix`